src/tbb/thread_data.h

```diff
@@ -145,6 +152,9 @@ class thread_data : public ::rml::job
     //! Indicates if the thread is created by RML
     const bool my_is_worker;
 
+    //! Is the slot occupied in arena begongs to workers' quota?
+    slot_type my_worker_slot_type;
```

Comment:
Seems `bool` would be enough. What about `my_is_workers_slot_occupied`?

Reply:
Done.